### PR TITLE
ROC-4403 - Add new pages and update task list ready for e2e for claimant respons…

### PR DESCRIPTION
**Pull request by ian-mcwilliams**
…e full admission by set date

### JIRA link

https://tools.hmcts.net/jira/browse/ROC-4403

### Change description

Page models added for:
How they want to pay (accept or reject defendant's plan)
Choose how to proceed (settlement or ccj)
Sign a settlement agreement
You've accepted the repayment plan (confirmation of claimant response submission)

Page model for claimant response task list updated to add new relevant links

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-accept-payment-method.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  radioYes: 'input[id=acceptyes]',
+  radioNo: 'input[id=acceptNo]',
+  submit: 'input[id=saveAndContinue]'
```
Comment (pragnesh-patel) [inline src/integration-test/tests/citizen/claimantResponse/pages/claimant-accept-payment-method.ts]: declare buttons outside of fields, as we do with other page objects
i.e 
`const buttons = {
      saveAndContinue: 'input[id=saveAndContinue]'
}`

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-accept-payment-method.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+export class ClaimantAcceptPaymentMethod {
+
+  chooseYes (): void {
+    I.checkOption(fields.radioYes)
+    I.click(fields.submit)
```
Comment (pragnesh-patel) [inline src/integration-test/tests/citizen/claimantResponse/pages/claimant-accept-payment-method.ts]: which would become 
I.click(buttons.saveAndContinue)

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  radioSettlement: 'input[id=optionsignSettlementAgreement]',
+  radioCcj: 'input[id=optionrequestCCJ]',
+  submit: 'input[id=saveAndContinue]'
```
Comment (pragnesh-patel) [inline src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts]: See above comments

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+export class ClaimantChooseHowToProceed {
+
+  chooseSettlement (): void {
+    I.checkOption(fields.radioSettlement)
+    I.click(fields.submit)
```
Comment (pragnesh-patel) [inline src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts]: see above

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  }
+
+  chooseCcj (): void {
+    I.checkOption(fields.radioCcj)
+    I.click(fields.submit)
```
Comment (pragnesh-patel) [inline src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts]: same

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-confirmation.ts [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  goToYourAccount: 'input[href=/dashboard]'
```
Comment (pragnesh-patel) [inline src/integration-test/tests/citizen/claimantResponse/pages/claimant-confirmation.ts]: naming : `linkGotoYourAccount` or `linkYourAccount`?

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-confirmation.ts [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+}
+
+export class ClaimantConfirmation {
+
+  clickGoToYourAccount (): void {
+    I.click(fields.goToYourAccount)
```
Comment (pragnesh-patel) [inline src/integration-test/tests/citizen/claimantResponse/pages/claimant-confirmation.ts]: Which would become `I.click(fields.linkGotoYourAccount)`

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-sign-settlement-agreement.ts [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  checkboxConfirm: 'input[id=signedTrue]',
+  submit: 'input[id=saveAndContinue]'
```
Comment (pragnesh-patel) [inline src/integration-test/tests/citizen/claimantResponse/pages/claimant-sign-settlement-agreement.ts]: see above comments for defining buttons for pageobjects outside of fields

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts (hunk 5 above)
Comment (pragnesh-patel) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts]: `I.checkOption(fields.radioCcj)` rename radio-button to maybe `I.checkOption(fields.radioRequestCcj)`

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-accept-payment-method.ts (hunk 1 above)
Comment (ian-mcwilliams) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-accept-payment-method.ts]: Done

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-accept-payment-method.ts (hunk 2 above)
Comment (ian-mcwilliams) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-accept-payment-method.ts]: Done

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts (hunk 3 above)
Comment (ian-mcwilliams) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts]: Done

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts (hunk 4 above)
Comment (ian-mcwilliams) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts]: Done

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts (hunk 5 above)
Comment (ian-mcwilliams) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts]: Done

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-confirmation.ts (hunk 6 above)
Comment (ian-mcwilliams) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-confirmation.ts]: Done

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-confirmation.ts (hunk 7 above)
Comment (ian-mcwilliams) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-confirmation.ts]: Done

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-sign-settlement-agreement.ts (hunk 8 above)
Comment (ian-mcwilliams) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-sign-settlement-agreement.ts]: Done

**Pull request merged**